### DISCUSSION TITLE
ant design 5使用 design token修改组件的UI，支持渐变背景色吗？

### DISCUSSION THREAD

**Discussion by ideaviewes** [ant-design/ant-design#42929] [Q&A] [answered]
后台他们设计师设计了一下，菜单的地方需要自定义，不知道能不能用design token来实现背景渐变的效果

**Comment by BoyYangzai** [comment, marked as answer]
@ideaviewes 
Hello
你说的是像这样子的整个 `menu` 的 `background` 吗 
https://stackblitz.com/edit/react-24mkzb?file=demo.tsx
如果不是的话 就要看对应的 `design token` 是否有覆盖 `background `
因为渐变色是要使用 `background 或者 background-image`去实现的 而 design token中一般只有 `colorBgContainer` 是直接放在 `background`下的 
所以 `menu` 中如果 `colorBgContainer` 不能满足你的需求的话 我推荐直接使用 `className` 自己加一个 `background样式` 去支持渐变色
@MadCcc

**Reply by ideaviewes** [reply]
好的，多谢

**Reply by vitalijalbu** [reply]
Would be fine to keep the `divider` on the `horizontal` layout @zombieJ
